showcase.json

```diff
@@ -3432,18 +3429,20 @@
   {
     "id": "gamebloc",
     "name": "Game Bloc",
-    "oneLiner": "A decentralised platform for hosting game tournaments which can either be crowdfunded or prepaid.",
+    "oneLiner": "A decentralized platform that lets gamers and gaming organizations create and join game tournaments.",
```

Comment:
Using _empower_ instead of _let_ feels more impactful and aspirational. Additionally, _manage_ might be a better fit than _create_, as it not only implies creation but also encompasses broader CRUD-like functionality (assuming that exists).

Suggestion:

A decentralized platform empowering gamers and gaming organizations to create, manage, and participate in game tournaments.